sonic-pcied/tests/test_pcied.py

```diff
@@ -18,6 +18,7 @@
 # Add mocked_libs path so that the file under test can load mocked modules from there
 mocked_libs_path = os.path.join(tests_path, "mocked_libs")
 sys.path.insert(0, mocked_libs_path)
+from sonic_py_common import daemon_base, device_info
```

Comment:
This line is same as L13. I think it will be ignored.
Do you want to remove L13?

Reply:
Fixed, this is a mistake.